google/cloud/video/transcoder_v1/types/resources.py

```diff
@@ -1345,7 +1345,7 @@ class AudioStream(proto.Message):
             -  ``sr`` - Side right channel
             -  ``fc`` - Front center channel
             -  ``lfe`` - Low frequency
-        mapping (Sequence[google.cloud.video.transcoder_v1.types.AudioStream.AudioMapping]):
+        mapping_ (Sequence[google.cloud.video.transcoder_v1.types.AudioStream.AudioMapping]):
```

Comment:
Need to revert this change

Reply:
See related PR in the generator: https://github.com/googleapis/gapic-generator-python/pull/1114

Reply:
Hi @parthea, could you please explain why you consider this a breaking change?
Per the PR you've mentioned `mapping` is a reserved filed name which is also used in Transcoder API proto. So right now AudioMapping and TextMapping classes cannot be used to configure stream mapping https://cloud.google.com/transcoder/docs/concepts/stream-mappings as it fails with the following error:
```
Traceback (most recent call last):
  File "/usr/src/app/./mapping_issue.py", line 8, in <module>
    text_stream=transcoder_v1.types.TextStream(
  File "/usr/local/lib/python3.9/site-packages/proto/message.py", line 496, in __init__
    raise TypeError(
TypeError: Invalid constructor input for TextStream: [atom_key: "atom0"
input_key: "srtEN"
]
```
This change will let the users to configure stream mapping via `mapping_` field similar to how the `type_` field is configured right now https://github.com/googleapis/python-video-transcoder/blob/main/google/cloud/video/transcoder_v1/types/resources.py#L367-L368

@software-dov for context

Reply:
This change is no longer breaking with the fix in `proto-plus>=1.20.1` to mitigate collisions in field names: https://github.com/googleapis/proto-plus-python/pull/295